todo_app/app.py

```diff
@@ -1,33 +1,34 @@
 from flask import Flask, render_template, request
 from todo_app.flask_config import Config
-import todo_app.data.session_items as fl
+from todo_app.data.session_items import *
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+def render_index_response():
+    return render_template('index.html', items = get_items())
 
 @app.route('/', methods=['GET', 'POST'])
```

Comment:
Something I didn't notice originally - this doesn't need to accept POST requests. You can make small fixes like this directly on master without creating a pull request.